snap/snapcraft.yaml

```diff
@@ -13,13 +13,17 @@ confinement: strict
 
 parts:
     novnc:
-        source: build/
+        source: https://github.com/novnc/noVNC/archive/v1.2.0.tar.gz
         plugin: dump
         stage-packages:
-            - websockify
             - bash
             - jq
-            - python-numpy
+            - python3-numpy
+            
+    websockify:
+        source: https://github.com/novnc/websockify/archive/v0.9.0.tar.gz
```

Comment:
Sounds good. But is there some special URL that always gives us the latest release? We'd like to avoid having to update this.

Reply:
Yes, you can easily do this:
`    websockify:
        source: https://github.com/novnc/websockify.git
        source-type: git
`

The only thing is then you're potentially getting a different version of websockify each time you build the snap, and that version may be a development version instead of a tested released version like 0.9.0.
You probably know more about the websockify project than me and can say whether this is a risk or not

Reply:
That's probably not a good idea. I was thinking the latest **release**, not the latest **commit**. So something like https://github.com/novnc/websockify/archive/latest.tar.gz. But if that doesn't exists I guess we'll have to live with updating that number.